File: exp/UnityChan2DTutorial/Assets/ScriptsCSharp/UI/CountTime.cs
using UnityEngine;
using UnityEngine.UI ;
using System.Collections;

public class CountTime : MonoBehaviour {
	public int maxTime = 120 ;
	int currentTime = 0 ;

	void Start () {
		currentTime = maxTime ;
		InvokeRepeating("CountDown", 0, 1) ;
	}
	
	void CountDown () {
		currentTime -= 1 ;
		GetComponent<Text>().text = "TIME\n" + currentTime ;
	}
}

File: exp/UnityChan2DTutorial/Assets/ScriptsJs/UI/CountTime.js
#pragma strict
var maxTime : int = 120 ;
private var currentTime : int = 0 ;

function Start(){
  currentTime = maxTime ;
  InvokeRepeating("CountDown", 0, 1) ;
}

function CountDown(){
  currentTime -= 1 ;
  GetComponent.<UI.Text>().text = "TIME\n" + currentTime ;
}